app/uk/gov/hmrc/customs/notification/controllers/HeaderValidator.scala

```diff
@@ -124,7 +124,7 @@ trait HeaderValidator {
   private def logValidationResult(headerName: => String, validationResult: => Boolean)(implicit h: Headers): Unit = {
     val resultText = if (validationResult) "passed" else "failed"
     val msg = s"$headerName header $resultText validation"
-    notificationLogger.debugWithHeaders(msg, h.headers)
+    notificationLogger.debugWithoutHeaders(msg, h.headers)
```

Comment:
why does debugWithoutHeaders have a headers param?

Reply:
The full set of headers is passed through so conversationId etc can be prefixed to the log message - hence the headers are required. The difference is that the headers are NOT them spat out verbatim into the message, hence the withoutHeaders in the name.